### ISSUE TITLE
Does this module supports general AWS API or ParameterStore?

### ISSUE THREAD

**Issue by mong0520** [Kong/lua-resty-aws#16]
Does this module supports general AWS API or ParameterStore?
Thank you.

**Comment by Tieske** [comment]
yes, see example here: https://kong.github.io/lua-resty-aws/classes/AWS.html#AWS:new is for SecretsManager, but should be easy to adapt to ParameterStore
